src/themes/default/pages/Product.vue

```diff
@@ -336,6 +326,9 @@ export default {
     },
     storeView () {
       return currentStoreView()
+    },
+    getJsonLd () {
+      return productJsonLd(this.getCurrentProduct, this.getCurrentProductConfiguration.color.label, this.getMaterials(this.getCurrentProduct.material), this.$store.state.storeView.i18n.currencyCode)
```

Comment:
one more thing :) Can you also move `getMaterials` from component to helper? It's needed only for `productJsonLd` and if you already pass `this.getCurrentProduct` as argument then you can get material in `productJsonLd`.
So this will be shorter:
`productJsonLd(this.getCurrentProduct, this.getCurrentProductConfiguration.color.label, this.$store.state.storeView.i18n.currencyCode)`

Reply:
@gibkigonzo you also need `this.getCustomAttributes` to `getMatrials`, so I'm not sure that it is good idea because I need to pass the same number of arguments, wdyt?